src/helpers/getByAPI.ts

```diff
@@ -87,7 +80,14 @@ export const UNSAFE_getByProps = (
     try {
       return instance.findByProps(props);
     } catch (error) {
-      throw new ErrorWithStack(prepareErrorMessage(error), getByPropsFn);
+      if (error instanceof Error) {
```

Comment:
I'd suggest make `prepareErrorMessage` accept `unknown` as first argument, and move the non-`Error` handling there.

Reply:
Yep, I've actually started doing that in other functions, I'll switch to this approach everywhere